src/postcapture.rs

```diff
@@ -118,7 +114,42 @@ impl PostCaptureAction for CopyToClipboard {
     }
 }
 
+/// Executes the post capture actions in the order they are defined in the settings.
+pub fn do_postcapture_actions(history_model: &HistoryModel, conn: &SqliteConnection, pixbuf: &mut Pixbuf) {
+    for action in get_actions_from_settings() {
+        action.handle(history_model, conn, pixbuf)
+    }
+}
+
+fn get_actions_from_settings() -> Vec<&'static dyn PostCaptureAction> {
+    let settings = gio::Settings::new("kc.kcshot");
+    let actions: Vec<String> = settings.strv("post-capture-actions")
```

Comment:
Could we name this `action_names`? Also do we really need to convert the `GString` to a `String`? We use `.as_str()` later on elements of the elements of these `Vec` and `GString` has that method too.